### PR TITLE
pomfcoin

### LINKED ISSUE #8589: Replace Goonecode toy in the Goonecode containment with 400 space credits
Alternatively, change it to 1 BTC :^)

### PR DESCRIPTION
**Pull request by DeityLink**
Fulfills #12700
Fixes #8589

Honestly why wasn't this a thing yet?

:cl:
* experiment: The goon satellite vault has a new unique reward in its display case. Though you can still find the goon code tossed on the floor beside it.

### CODE REVIEW

**Comment by Eneocho** [comment]
So, what do we pay the gods in exchange for wishes, the pomfcoin or the goonecode?

**Comment by Durgen** [comment]
>Fewer still can make use of it.

is that a reference to prayers being mostly ignored

**Comment by DeityLink** [comment]
> 
> 
> > Fewer still can make use of it.
> 
> is that a reference to prayers being mostly ignored

sure is. (honestly most prayers I've answered recently, the player fucked off do something else while I was in the process of working my ass to make something nice for them.)

File: code/game/objects/structures/displaycase.dm [hunk 1]
```diff
@@ -44,8 +44,7 @@
 	update_icon()
 
 /obj/structure/displaycase/gooncode/setContent()
-	occupant=new /obj/item/toy/gooncode(src)
-	desc = "The glass is cracked and there are traces of something leaking out."
```
Comment (Kurfursten) [inline code/game/objects/structures/displaycase.dm:48]: Should change this to "It looks like something has fully leaked out." now

File: code/modules/mining/ores_coins.dm [hunk 2]
```diff
@@ -564,3 +564,29 @@
...
+	credits = 2525
+	siemens_coefficient = 1
+	melt_temperature=1768+T0C
+	force = 4
+	throwforce = 4
+
+/obj/item/weapon/coin/pumf
+	material="pumf"
+	name = "pumf coin"
+	desc = "A slade coin featuring the effigy of an angry chicken. If you come into its possession that means you've been a naughty boy, whatever you've been doing stop it."
```
Comment (Kurfursten) [inline code/modules/mining/ores_coins.dm]: `"if it comes into your possession"` is proper in English
Also, the back end of this sentence needs to be separated as it is an independent phrase:
`"...you've been a naughty boy. Whatever you've been doing, stop it."`

File: code/game/objects/structures/displaycase.dm (hunk 1 above)
Comment (DeityLink) [reply code/game/objects/structures/displaycase.dm:48]: I'll put that description directly on the gooncode, the case itself now contains the pomfcoin

**Comment by DeityLink** [comment]
there

**Pull request merged**